scanpy/readwrite.py

```diff
@@ -277,15 +277,49 @@ def _read_v3_10x_h5(filename, *, start=None):
                 (data, dsets['indices'], dsets['indptr']),
                 shape=(N, M),
             )
+            obs_dict = dict(obs_names=dsets['barcodes'].astype(str))
+            var_dict = dict(
+                var_names=dsets['name'].astype(str),
+            )
+            if 'gene_id' not in dsets:
+                var_dict.update(
+                    {
+                        'gene_ids': dsets['id'].astype(str),
+                        'feature_types': dsets['feature_type'].astype(str),
+                    }
+                )
+            else:
+                var_dict.update(
+                    {
+                        'gene_ids': dsets['gene_id'].astype(str),
+                        'id': dsets['id'].astype(str),
```

Comment:
```suggestion
                        'probe_ids': dsets['id'].astype(str),
```
I'm inclined to name this `probe_ids` to contrast it with `gene_ids`.

Reply:
I would prefer `probe_id`, but they use `gene_ids` not `gene_id` (which does not make sense to me for columns in a dataframe). Made the change.